src/ucs/debug/instrument.h

```diff
@@ -117,5 +117,16 @@ void __ucs_instrument_record(uint64_t location, uint64_t lparam, uint32_t wparam
 
 #endif
 
+#if ENABLE_IB_TX_INSTRUMENTATION
+#define UCS_INSTRUMENT_IB_TX_RECORD UCS_INSTRUMENT_RECORD
```

Comment:
in release version instrumentation is disabled anyway (see definition of UCS_INSTRUMENT_RECORD)